paasta_tools/cli/cmds/security_check.py

```diff
@@ -45,14 +46,22 @@ def add_subparser(subparsers):
 
 
 def perform_security_check(args):
+    """It runs a few security-checks including:
+    - shellshock: is the version of bash running inside the container vulnerable to shellshock?
+    - docker container: is the container running as root?
+    - packages: does the container has the latest packages installed? If not, the output will contain the name of the
+    package that needs to be updated and to which version.
```

Comment:
Yes, that's true. I made the docstrings verbose for the readthedocs.

But hope it's better now. :)